gcc/testsuite/rust/execute/torture/inline_asm_mov_x_5_ARM.rs

```diff
@@ -13,12 +13,24 @@ extern "C" {
 
 fn main() -> i32 {
     let mut _x: i32 = 0;
+    let mut _y: i32 = 9;
```

Comment:
any reason why you're using underscore prefixed variables ? Correct me if I'm wrong but the compiler should not emit any warning since we're using `_x` in the print call.

For `_y` we might be missing some things though since it is only used in the assembly.

Reply:
Oops this is me copying from my previous test suites. It's just to help the dev cycle. I'll change it asap